packages/react-redux/src/slices/cartoSlice.d.ts

```diff
@@ -104,7 +105,7 @@ export function setBasemap(
 export function addFilter(
   filter: Filter
 ): {
-  type: CartoActions.ADD_LAYER;
+  type: CartoActions.ADD_FILTER;
```

Comment:
Was this a BUG??. ⚠️

Reply:
Yes, but it's related to typescript. Does not affect functionality